amethyst_renderer/Cargo.toml

```diff
@@ -25,19 +25,24 @@ opengl = ["gfx_device_gl", "gfx_window_glutin", "glutin"]
 vulkan = ["gfx_device_vulkan", "gfx_window_vulkan"]
 
 [dependencies]
-cgmath = { version = "0.14", features = ["eders", "use_simd"] }
+cgmath = { version = "0.14", features = ["eders"] }
+derivative = "1.0"
 fnv = "1.0"
-gfx = { version = "0.15", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
-gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+#gfx = { version = "0.15", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+gfx = { version = "0.15", features = ["cgmath-types", "serialize"], path = "/home/ekalderon/gfx/src/render/" }
+#gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], path = "/home/ekalderon/gfx/src/core/" }
 gfx_macros = "0.2"
 num_cpus = "1.0"
 rayon = "0.7"
 serde = "1.0"
 serde_derive = "1.0"
 winit = "0.6"
 
-gfx_device_gl = { version = "0.14", optional = true, git = "https://github.com/gfx-rs/gfx" }
-gfx_window_glutin = { version = "0.16", optional = true, git = "https://github.com/gfx-rs/gfx" }
+#gfx_device_gl = { version = "0.14", optional = true, git = "https://github.com/gfx-rs/gfx" }
+gfx_device_gl = { version = "0.14", optional = true, path = "/home/ekalderon/gfx/src/backend/gl/" }
```

Comment:
perhaps, reconsider including `gfx_device_gl` too. I believe `gfx_window_glutin` now re-exports the essentials